doc/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/managing-iot-data.md

```diff
@@ -1 +1,404 @@
-# To be replaced with content
+# Managing IoT data
+With the Tour Guide application, you can generate, simulate and give persistence to data coming from IoT devices.  For that purpose, we use the information of restaurants, and
```

Comment:
we generate humidity and temperature (virtual) sensors for the kitchen and dining room of restaurants.